casas_gis_old/scripts/subtract.sh

```diff
@@ -79,4 +83,4 @@ if [ -n "$GIS_OPT_diffFile" ] ; then
 	diffFile=$GIS_OPT_diffFile
 fi
 
-perl ~/SubtractOutputGui.pl "$workDir" "$warmfile" "$obsFile" "$diffFile"
+perl ../PerlScripts/SubtractOutputGui.pl "$workDir" "$warmfile" "$obsFile" "$diffFile"
```

Comment:
Probably `SubtractOutputGui.pl` was never developed and SubtractOutput.pl continued to be used instead.

`SubtractOutput.pl` now included in the `scripts` dir